src/t8_forest/t8_forest_vtk.cxx

```diff
@@ -334,13 +334,13 @@ t8_curved_element_get_reference_node_coords (const t8_element_t *elem,
 #endif
 
 int
-t8_forest_write_vtk_via_API (t8_forest_t forest, const char *fileprefix,
-                             int write_treeid,
-                             int write_mpirank,
-                             int write_level,
-                             int write_element_id,
-                             int curved_flag,
-                             int num_data, t8_vtk_data_field_t *data)
+t8_forest_vtk_write_file_via_API (t8_forest_t forest, const char *fileprefix,
+                                  int write_treeid,
+                                  int write_mpirank,
+                                  int write_level,
+                                  int write_element_id,
+                                  int curved_flag,
+                                  int num_data, t8_vtk_data_field_t * data)
```

Comment:
Please add the t8_vtk_data_field_t type to the list of special t8code data type for indent (see the `scripts/` subfolder).
Every custom type should indent as `TYPE *name`  not `TYPE * name`.